client/lib/utils/pdf-support.js

```diff
@@ -12,6 +12,10 @@ import _ from 'lodash';
  * PDFs at all.
  */
 export default _.memoize( () => {
+	if ( /iPad|iPhone|iPod/.test( navigator.userAgent ) && ! window.MSStream ) {
```

Comment:
Damn those silly user-agents! I've just learnt that IE Mobile uses `like iPhone` on its user-agent, thats low :(

Could you please add a comment about why this is needed? This file is full of seemingly arbitrary rules that should be explained. Then this is good to go :)

Reply:
@DanReyLop - I added a commit containing documentation. Thanks for the review!